pubmedcentral/to-dolma.py

```diff
@@ -0,0 +1,74 @@
+import argparse
+import datetime
+import functools
+import os
+
+from licensed_pile.licenses import PermissiveLicenses
+from licensed_pile.write import to_dolma
+
+parser = argparse.ArgumentParser(description="Collect PubMedCentral into Dolma format.")
+parser.add_argument("--filelist", help="The path to the filelist.txt file.")
+parser.add_argument(
+    "--data_dir", default="data/raw/", help="Path to the directory of markdown files."
+)
+parser.add_argument(
+    "--output_dir",
+    default="data/dolma/",
+    help="Where the dolma formatted data goes.",
+)
+parser.add_argument(
+    "--filename",
+    default="pubmedcentral.jsonl.gz",
+    help="The base filename for our books.",
+)
+parser.add_argument(
+    "--shard_size", type=int, default=1, help="Size, in GB, for each shard."
+)
+
+
+LICENSE_MAP = {
+    "CC0": PermissiveLicenses.CC0,
+    "CC BY": PermissiveLicenses.CC_BY,
+    "CC BY-SA": PermissiveLicenses.CC_BY_SA,
+}
+SOURCE_NAME = "PubMed Central"
+
+
+def format_dolma(example: str, data_dir: str, source_name: str = SOURCE_NAME):
+    file, citation, accessionID, PMID, lic = example.split("\t")
```

Comment:
What is the difference between `accessionID` and `PMID`? Should we save the `PMID` into the metadata field? If we aren't can we replace it with `_,` as we aren't using it?